daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/value/ValueCoderSpec.scala

```diff
@@ -189,10 +189,39 @@ class ValueCoderSpec
     Right(value) shouldEqual fromToBytes
   }
 
+  def normalize(value0: Value[ContractId], version: TransactionVersion): Value[ContractId] = {
```

Comment:
Given that you called `enrich` normalize yesterday and this one does the opposite maybe go for something more descriptive like `stripTypes` or `stripMetadata` to make it explicit what it does :)